app/ui/src/app/customizations/api-connector/api-connector.service.ts

```diff
@@ -2,7 +2,7 @@ import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs/Observable';
 import { Restangular } from 'ngx-restangular';
 
-import { RESTService } from '../../store/entity/rest.service';
+import { RESTService } from '../../store/entity';
```

Comment:
Hey! Now you can use aliases here like:

`import { RESTService } from '@syndesis/ui/store/entity';`

This way, you can skip refactoring the import paths if you ever need to move `api-connector.service.ts` elsewhere.

Reply:
Cool, even better!

Reply:
In fact, the entity sub-folder features its own barrel, so you can shorten all this as:

`import { RESTService } from '@syndesis/ui/store`